robocop/checkers/naming.py

```diff
@@ -1094,7 +1094,7 @@ def visit_Var(self, node):  # noqa
         self.check_for_reserved_naming_or_hyphen(variable, "Variable", is_assign=True)
         # TODO Check supported syntax for variable, ie ${{var}}?
         if not self._is_var_scope_local(node):
-            self.check_non_local_variable(variable.value, node, variable)
+            self.check_non_local_variable(search_variable(variable.value).base, node, variable)
```

Comment:
search_variable may fail if the variable is invalid (for example not closed properly {}). We can just wrap it with try except as we did it in visit_Variable in the same file to safeguard it

Reply:
Ok nvm, we have return if node contains any error just few lines before - it should cover it

Reply:
It's that covered by the returns in lines 1090 and 1093?